packages/docusaurus-theme-classic/src/index.js

```diff
@@ -12,7 +12,7 @@ const path = require('path');
 const storageKey = 'theme';
 const noFlash = `(function() {
   function setDataThemeAttribute(theme) {
-    document.querySelector('html').setAttribute('data-theme', theme);
+    document.documentElement.setAttribute('data-theme', theme);
```

Comment:
For sure this is faster, it's direct access. But fewer people have heard about this lol. It's fine since it's internal.

Reply:
i think it is fine because relevant people can simply google and learn what this is

Reply:
Yes I agree it's fine :)